options/options.js

```diff
@@ -1,26 +1,26 @@
 "use strict";
 
 $(() => {
-  $('.message a').click(function(e){
+  $('.message a').click(e => {
     $('.error').hide();
     $('.login-container').animate({height: 'hide', opacity: 'hide'}, 'slow');
     $(`.${e.target.name}-login-container`).animate({height: 'show', opacity: 'show'}, 'slow');
   });
-  $('#login').click((e) => {
+  $('#login').click(e => {
     addCred(getGithubParam());
   });
-  $('#ghe-login').click((e) => {
+  $('#ghe-login').click(e => {
     addCred(getGHEParam());
   });
-  $('#bitbucket-login').click((e) => {
+  $('#bitbucket-login').click(e => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/3bad4bc025cc43cc251f17b68f22b94ac54ac1f0/options/options.js#L15"></span>‘e’ is defined but never used.</td>
    </tr>
  </tbody>
</table>